packages/core/src/components/modal-content/modal-content.scss

```diff
@@ -15,4 +15,5 @@
   position: relative;
   overflow: auto;
   padding: 0.125rem 0.625rem 0.125rem 0.625rem;
+  max-content: 60vh;
```

Comment:
Not a valid CSS property? See e.g. [max-content](https://developer.mozilla.org/en-US/docs/Web/CSS/max-content).